recipes/graylog-forwarder/files/post-install

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+set -e
+
+user="graylog-forwarder"
+group="graylog-forwarder"
+datadir="/usr/share/graylog-forwarder"
+cfgdir="/etc/graylog/forwarder"
+journaldir="/var/lib/graylog-forwarder"
```

Comment:
This can be removed once we set the `datadir` above.

Reply:
Done.